Source/Dafny/Resolver.cs

```diff
@@ -16942,11 +16945,12 @@ class MethodCallInformation {
           actuals.Add(b.Actual);
           substMap.Add(formal, b.Actual);
           var what = whatKind + (context is Method ? " in-parameter" : " argument");
-          if (formals.Count != 1) {
-            what += " " + j;
+          var displayName = formal.HasName && !(formal is ImplicitFormal);
+          if (!namesProvidedExplicitly.Contains(formal.Name) && formals.Count() > 1) {
```

Comment:
I would also be okay with not having the `namesProvidedExplicitly` logic and always showing the index.

Reply:
Me too and that would be my preference, so I removed that logic.